rel-eng/tito.props

```diff
@@ -320,12 +320,11 @@ whitelist = foreman
   rubygem-spice-html5-rails
   rubygem-sprockets
   rubygem-sprockets-rails
+  rubygem-sqlite3
   rubygem-sshkey
   rubygem-statsd-instrument
-  rubygem-sqlite3
   rubygem-syntax
   rubygem-text
-  rubygem-tilt
```

Comment:
Is it correct this one is deleted? Everything else just got reordered, this one is removed though :thinking:

Reply:
yes, it was a dependency of sass-rails 5.0

Reply:
actually should probably be removed from the manifest and comps files as well